lib/BloqadeSchema/schema_examples/aws_science_tasks/global_rydberg/global5.jl

```diff
@@ -27,35 +46,64 @@ end
 
 
 # task 2
+# X[pi/2] pulse, wait T, Y[pi/2] pulse
 
 atoms = AtomList([(0.0, 0.0)])
 
 T_pi_2 = 1/16
 T= 0.1
-epsilon =0.05
+# epsilon =0.05
+#= 
 Ω = piecewise_constant(clocks=[0.0, epsilon, T_pi_2+ epsilon, T + T_pi_2+epsilon, T + 2* T_pi_2+epsilon, T + 2* T_pi_2+2* epsilon], values= 2π*[0.0, 4, 0, 4, 0])
 ϕ = piecewise_constant(clocks=[0.0, epsilon, T_pi_2+ epsilon, T + T_pi_2+epsilon, T + 2* T_pi_2+epsilon, T + 2* T_pi_2+2* epsilon], values= 2π*[0.0, 0, 0, 0, 0])
+=#
+Ω = piecewise_constant(clocks = [0.0, T_pi_2, T + T_pi_2, T + 2 * T_pi_2], values = 2π * [4,0,4] )
+ϕ = constant(;duration = T + 2 * T_pi_2, value = 0)
```

Comment:
```suggestion
ϕ = piecewise_constant(clocks = [0.0, T_pi_2, T + T_pi_2, T + 2 * T_pi_2], values =  [0.0,0.0,π/2] )
```

looks like the example was wrong, good catch @johnzl-777 ! this adds a pulse in the phase to shift from X to Y, @johnzl-777 let me know if this looks correct to you. Also once you commit this make sure to re-run the script to generate the JSON file.